web/src/hooks/usePolling.test.tsx

```diff
@@ -0,0 +1,10 @@
+import {renderHook} from '@testing-library/react-hooks';
+import usePolling from './usePolling';
+
+test('usePolling', async () => {
+  const callback = jest.fn();
+  const payload = {callback, isPolling: true, delay: 0};
+  await renderHook(() => usePolling(payload));
+
+  expect(callback).toBeCalledTimes(0);
```

Comment:
Shouldn't the tests be more extensive to cover the different scenarios, like when the `isPolling` could change, etc?

Reply:
i am focusing on getting test suites present. 
testing setInterval is a bit tricky.
I am aiming on getting coverage up on those 3 folders.